contents/blog/how-we-built-email-onboarding.md

```diff
@@ -0,0 +1,129 @@
+
+## Onboarding 4.0: The one where we filtered by ICP
+
+[4.0](https://github.com/PostHog/meta/pull/150/) was a massive step up in complexity as a result of our growing number of features. By late 2023 the flow had 28 separate emails, of which users would get a maximum of 6 over several weeks. We still worried this would be too many, but the low (0.4%) unsubscribe rate suggested otherwise.
+
+![Onboarding 4.0](../images/blog/onboarding/onboarding_31.png)
+<Caption>Onboarding 4.0 split high-ICP users out into their own email flow because our CS team would reach out directly</Caption>
+
+By this point we’d also changed our conversion event to mean a user enabling billing for a new product within one week of opening an email. 
+
+- **Open rate:** 52%
+- **Click-through-rate:** 4%
+- **Unsubscribe rate:** 0.4%
+- **Conversion rate:** 6%
+
+Around this point we also started filtering out teams in our [ideal customer profile](/newsletter/ideal-customer-profile-framework) (ICP) using a scoring system implemented in Hubspot. If a user scored over a certain threshold they’d be moved out of the usual flow so [our customer success team](/handbook/small-teams/customer-success) could reach out directly. We’ve since removed this check because it felt too salesy and didn’t perform well, however. 
+
+## Onboarding 5.0: The current one
+
+At the time of writing this is the current onboarding flow. It contains over 53 emails and 38 timed events and was, quite simply, a pain in the ass to explain to everyone internally.
+
+The goal of this flow is to focus on improving activation, and build on some parallel improvements to our in-app onboarding flow. Now when a user signs up to PostHog we ask them to select a product to setup first: analytics, replays, flags, or surveys. [Onboarding 5.0](https://github.com/PostHog/meta/issues/172) builds off that selection by encouraging users to use that product, and then showing how other features can combine with it.
+
+![Onboarding 5.0](../images/blog/onboarding/onboarding_31.png)
+<Caption>Onboarding 5.0: Messy, complex, and highly personalized to user interests</Caption>
+
+The hope was that this would improve in-app activation, which we roughly define as: 
+
+**Product analytics:** Events ingested, 1 insight created, 1 dashboard created
+**Session replay:** 5 recordings viewed
+**Feature flags:** 1 feature flag created
+**Surveys:** 1 survey launched
+
+To avoid annoying users, we added a time block based on our activation criteria – if a user hasn’t activated after two emails, we stop sending them emails for six weeks. We also use an unsubscribe notice that tells users how many emails they can typically expect: one email per week, for six weeks. 
+
+## What’s next?
```

Comment:
I think you missed my comment about this, but could you add a section before "What's next?" that just summarizes what you learned through all these revisions. It just needs some takeaways for the reader to remember.

Reply:
```suggestion
## What we learned

Across the five major versions (and many other smaller iterations) of our email onboarding, there are a few definite things we've learned. 

### Email works for engineers too
As I said at the start, the conventional wisdom is that engineers are particularly skeptical of any marketing activity, and likely to unsubscribe to emails. However, our unsubscribe rates have been consistently low enough to disprove this. If the emails add value for engineers and are [well-written for that audience](/founders/writing-for-developers), they can still be effective. 

### You can send more emails than you think
One of our consistent worries has been that we send too many emails. Again though, the low unsubscribe rate suggests that users aren't getting annoyed by this. The current email flow gives users approximately six emails, yet we've not heard any complaints about this. 

We think the reason for this is...

1. We space emails out quite conservatively. Users will typically only get one email per week, and we are defensive about deviating from this schedule. 
2. We tell users how many emails they'll get. It's added to the welcome email, right next to the unsubscribe button. This means users know we value their time and can unsubscribe _before_ they get annoyed. 
3. Our emails are actually worth reading. We don't send those awful outreach emails from sales reps, we don't run limited-time discounts, and we don't beg users to pay more. We just tell them how to get more out of PostHog. 

### Users prefer fun visual art to screenshots
I mean, who _wouldn't_ want to look at a cute hedgehog instead of a graph? 

We've run tests on this several times and [Lottie's](https://posthog.com/community/profiles/227) art is victorious by a small margin every time. 

### Personalization matters
Personalization doesn't mean calling users by their first name using a [liquid](https://customer.io/docs/journeys/using-liquid/) tag and pretending to be best buds. It means providing the right content, at the right time, in the right way. 

We do this in several ways in the latest version - delaying our welcome email, tailoring the emails users receive based on their choices, and sending newsletter invites from my personal address. If nothing else, the replies I frequently receive from users are a testament to how effective this is. 

## What’s next?
```